.github/workflows/publish-template.yml

```diff
@@ -77,8 +77,17 @@ jobs:
       - name: Remove .npmrc file
         run: rm .npmrc
 
+      - name: Install Release Scripts dependencies
+        working-directory: ./release-scripts
+        run: npm install
+
       - name: Check npm for package availability
-        run: node ../../release-scripts/checkPackageAvailability.js ${{ inputs.path }}/${{ inputs.lib-name }}
+        working-directory: ./release-scripts
+        run: node checkPackageAvailability.js ${{ inputs.path }}/${{ inputs.lib-name }}
 
       - name: Create Github releases, discussions and milestones
-        run: node ../../.github/create-releases.js -lib ${{ inputs.path }}/${{ inputs.lib-name }}
+        working-directory: ./release-scripts
```

Comment:
Is there an option to set the working-directory across these three steps?

Reply:
Unfortunately no. There's a job level `working-directory` which is already set to something else for the rest of the steps in the job